admin/provisioner/provisioner.go

```diff
@@ -28,10 +28,17 @@ type ProvisionOptions struct {
 	GithubURL            string
 	GitBranch            string
 	GithubInstallationID int64
+	Envs                 map[string]string
+}
+
+// todo :: find a better name
+type UpdateProvisionOptions struct {
+	Envs map[string]string
 }
```

Comment:
I think the name is fine (doing `FuncNameOptions` is fairly idiomatic). But I think the recommendation is to add an `Options` type only when a param list grows long.